src/tetris/game.ts

```diff
@@ -4,9 +4,10 @@ import "phaser";
 import MainScene from "tetris/scene/mainScene";
 import BiasEngine from "tetris/biasEngine/biasEngine"
 import Profiler from "tetris/profiler/profiler";
-import "./styles/scss/styles.scss"
+// import "tetris/styles/scss/styles.scss"
 
-const biasEngine = new BiasEngine();
+const profiler = new Profiler();
```

Comment:
Instantiating new Objects should be done inside the game's constructor.

Reply:
Unfortunately, this has to be done this way since `profiler` is required for `biasEngine` which again is required for the `GameScene` ...